tests/testthat/test_build_lm.R

```diff
@@ -172,6 +172,15 @@ test_that("prediction with glm family (binomial) and link (probit) with target c
   test_data <- test_data %>% mutate(CARRIER = factor(CARRIER, ordered=TRUE)) # test handling of ordered factor
 
   model_data <- build_lm.fast(test_data, `CANCELLED X`, `logical col`, `Carrier Name`, CARRIER, DISTANCE, predictor_n = 3, model_type = "glm", family = "poisson", link = "log")
+  ret <- model_data %>% broom::glance(model)
```

Comment:
What are those tests (175-177) for? Maybe better to do some null check on each result?

Reply:
For now it's just exercising code to make sure there is no error.